packages/cli-tools/src/throwIfNonAllowedProtocol.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Check if a url uses an allowed protocol
+ */
+
+const ALLOWED_PROTOCOLS = new Set(['http:', 'https:', 'flipper:']);
+
+export default function throwIfNonAllowedProtocol(url: string) {
+  const _url = new URL(url);
+  const urlProtocol = _url.protocol;
+
+  if (!ALLOWED_PROTOCOLS.has(urlProtocol)) {
```

Comment:
Nit: You can also have an array and use `includes` here. `Set` is a bit overkill for this use case.